GliaWidgets/Public/Glia/Glia+Logging.swift

```diff
@@ -0,0 +1,88 @@
+extension Glia {
```

Comment:
This structures are not public, so let's move this file to another directory please.

Reply:
Moved it 1de9cac1347aebd466b6bf8a392de67800d6b4c2.